link_palindrome.c

```diff
@@ -0,0 +1,57 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <assert.h>
+#include <stdbool.h>
+#include "link_palindrome.h"
+
+void Check_Input(List* head)
+{
+	assert(head != NULL);
+	return;
+}
+
+int Get_Link_List_Length(List* head)
+{
+	int length = 0; 
+	List* temp = head; 
```

Comment:
it is more common to use the name "current" to refer to the element as you traverse the list